composer/workflows/airflow_db_cleanup.py

```diff
@@ -53,11 +53,15 @@
 
```

Comment:
Do we have test for Airflow 1 for this file (as this file will be for Airflow 1 as well)?

Reply:
If it's not in the airflow_1_samples directory, we're not properly testing dag parsing for Airflow 1 - if you want to support Airflow 1, leave the script in the other directory too even if it's only slight differences. The Airflow 1 directory has different setup/teardown code that's Airflow 1 specific

Reply:
I added separate Airflow 1 file and test